### PR TITLE
Update to 1.9 Sylius version

### PR DESCRIPTION
**Pull request by daniel8702**

### CODE REVIEW

**Comment by Sander-Voogt** [comment]
Is there date when this plugin will be supported?

**Comment by daniel8702** [comment]
Hi @Sander-Voogt, thanks for Your message.

We will update to the latest version of the Sylius this week or begin in the next week.

**Comment by acaenepeel** [comment]
Is there an update on when this plugin will support Sylius 1.9?

File: composer.json [hunk 1]
```diff
@@ -6,8 +6,9 @@
     "require": {
         "php": "^7.3",
         "sylius/sylius": "^1.8",
+        "symfony/messenger": "^4.4 || ^5.2",
         "mollie/mollie-api-php": "^2.0",
-        "sylius/refund-plugin": "1.0.0-RC.5",
+        "sylius/refund-plugin": "1.0.0-RC.8",
         "sylius/admin-order-creation-plugin": "^0.9.0",
```
Comment (TheGrimmChester) [inline composer.json:12]: Hello, could you also update this requirement to ^0.11? Sylius updated their plugin to support v1.9 at this tag.
https://github.com/Sylius/AdminOrderCreationPlugin/releases/tag/v0.11.0

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -143,7 +162,7 @@ jobs:
 
       -
         name: Run Behat
-#        run: vendor/bin/behat --colors --strict -vvv --no-interaction || vendor/bin/behat --colors --strict -vvv --no-interaction --rerun
+#        run: vendor/bin/behat --colors --strict -vvv --no-interaction  -f progress || vendor/bin/behat --colors --strict -vvv --no-interaction --rerun
```
Comment (patrick477) [inline .github/workflows/build.yml:165]: :)

File: src/Resources/config/services/repository/credit_memo.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <services>
+        <service id="bitbag_sylius_mollie_plugin.repository.credit_memo_repository" class="BitBag\SyliusMolliePlugin\Repository\CreditMemoRepository" public="true" decorates="sylius_refund.repository.credit_memo">
+            <argument type="service" id="sylius_refund.manager.credit_memo"/>
+            <argument type="service">
+                <service class="Doctrine\ORM\Mapping\ClassMetadata" public="false">
+                    <argument>%sylius_refund.model.credit_memo.class%</argument>
+                </service>
+            </argument>
+        </service>
+    </services>
```
Comment (patrick477) [inline src/Resources/config/services/repository/credit_memo.xml:14]: then you can add your repo which will have parent `sylius_refund.repository.credit_memo`

**Review by patrick477 — APPROVED** [review]

**Pull request merged**